### PR TITLE
Add a code of conduct

### PR DESCRIPTION
**Pull request by lazka**
See https://github.com/msys2/MINGW-packages/issues/8597 for context

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -92,4 +92,5 @@ nav:
         - wiki/Sudo.md
         - wiki/Terminals.md
         - wiki/JIT-Debugging.md
-    - 'contact.md'
\ No newline at end of file
+    - 'contact.md'
+    - 'codeofconduct.md'
```
Comment (naveen521kk) [inline mkdocs.yml:96]: A new line at the end of file :)
```suggestion
    - 'codeofconduct.md'

```

**Review by naveen521kk — APPROVED** [review]
lgtm

**Comment by naveen521kk** [comment]
next, we should create a `.github` repository add a file called `CODE_OF_CONDUCT.md` with our code of conduct https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#creating-a-repository-for-default-files.

**Comment by lazka** [comment]
yeah, I thought just link to https://www.msys2.org/codeofconduct/ to keep it simple

**Pull request merged**